### PR TITLE
OWscDataSets: include information about an organism in data sets

### PR DESCRIPTION
**Pull request by JakaKokosar**
##### Issue
https://github.com/biolab/orange3-single-cell/issues/121

##### Description of changes
Depends on https://github.com/biolab/orange3/pull/2950

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: orangecontrib/single_cell/widgets/owscdatasets.py [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+import Orange.widgets.data.owdatasets
+
 
 from AnyQt.QtWidgets import QApplication
+from AnyQt.QtGui import QStandardItemModel, QStandardItem
+from AnyQt.QtCore import Qt
 
-import Orange.widgets.data.owdatasets
+from orangecontrib.bioinformatics.ncbi.taxonomy import shortname, name, other_names, common_taxids
+from orangecontrib.bioinformatics.widgets.utils.data import GENE_NAME, TAX_ID
```
Comment (lanzagar) [inline orangecontrib/single_cell/widgets/owscdatasets.py]: Some things are imported and never used.
But more importantly, we should not assume orangecontrib.bioinformatics is available.
It is not present in the requirements of this add-on, and even installing Bioinformatics through Orange does not provide this as it installs a different version (orangecontrib.bio).

Comment (BlazZupan) [reply orangecontrib/single_cell/widgets/owscdatasets.py]: This is ok, @JakaKokosar please add bioinformatics to requirements.

File: orangecontrib/single_cell/widgets/owscdatasets.py [hunk 2]
```diff
@@ -14,6 +19,76 @@ class OWscDataSets(Orange.widgets.data.owdatasets.OWDataSets):
...
+        ['taxid',       {'label': 'Organism'}],
+        ['target',      {'label': 'Target'}],
+        ['tags',        {'label': 'Tags'}]
+    ]
+
+    def load_data(self, path):
+        info = self.selected_dataset()
+        data = Orange.data.Table(path)
+        data.attributes[TAX_ID] = info.taxid
+        data.attributes[GENE_AS_ATTRIBUTE_NAME] = True  # Will all data sets have gene names in columns?
```
Comment (lanzagar) [inline orangecontrib/single_cell/widgets/owscdatasets.py]: No.

**Comment by lanzagar** [comment]
I am merging this, but a future PR should address two more issues:
- marker gene datasets to be removed from the repository (and migrated into Marker Genes if needed)
- num. of instances being shown as num. of cells is OK, variables however should contain the number of all columns, which depending on the number of metas and class vars is not the number of genes. Probably the json descriptions should include both (variables = num. of columns; genes = manually specified/curated num of actual genes)

**Pull request merged**